ocean-client/src/utils/store_aws.ts

```diff
@@ -34,7 +34,7 @@ export class StoreAWS implements IStore {
     }
 
     async clearSkip(): Promise<void> {
-        const key = StoreKey.Skip.replace("-maxi", "-maxi" + this.settings.paramPostFix)
+        const key = realParamKey(StoreKey.Skip.replace("-maxi", "-maxi" + this.settings.paramPostFix)
```

Comment:
does this even compile? there is a `)` missing. and duplication of that replace outside & inside of `realParamKey`